TehanovAnton/hometask-6/scrapper/parser.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+# Site parser
```

Comment:
Class comment should tell why this class was added.
What is the purpose for it being added?